package/yast2-network.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Thu May 25 09:27:09 UTC 2017 - mfilka@suse.com
+
+- bnc#1039532
+  - do not write hostname into /etc/hosts duplicitly for one IP
+  - configure static IPs to resolve to system wide hostname during
+    installation
```

Comment:
e.g. you could assign ```hostname1``` to ip 1.1.1.1 when editing particular device and ```hostname2``` on Hostname/DNS tab and the result was ```1.1.1.1 hostname1 hostname2```. After this PR it is ```1.1.1.1 hostname1```